packages/shared/src/hooks/useCookieBanner.ts

```diff
@@ -12,9 +13,12 @@ export function useCookieBanner(): [
 
   const acceptCookies = (): void => {
     setShowCookie(false);
-    document.cookie = `${consentCookieName}=true;path=/;domain=${
-      process.env.NEXT_PUBLIC_DOMAIN
-    };samesite=lax;expires=${60 * 60 * 24 * 365 * 10}`;
+    setCookie(consentCookieName, true, {
+      path: '/',
+      domain: process.env.NEXT_PUBLIC_DOMAIN,
+      sameSite: 'lax',
+      maxAge: 60 * 60 * 24 * 365 * 10,
+    });
```

Comment:
I think we had a bug in our code where our cookie banner `ilikecookies` cookie was always a session cookie. 

This was due to `expires` being set as `number` of seconds and per spec it should be date formatted string. 

I replaced it with `max-age` which correctly sets expiration of cookie in seconds (in this case 10 years). 

We did not notice this because nowadays browsers do not clear session cookies until their process is closed in full vs closing a tab.

Reply:
Amazing. Thanks for the retro fix. I think we can introduce a variable named `const TEN_YEARS` or something similar, to make easier to understand in one look.

Reply:
@rebelchris @sshanzel one fun fact for Chrome https://developer.chrome.com/blog/cookie-max-age-expires/#:~:text=With%20this%20change%2C%20Chrome%20caps,set%20to%20400%20days%20instead

I think we can still leave 10 years for other browsers but something to keep in mind.